samples/basic-order-processing-contract/src/integration.spec.ts

```diff
@@ -0,0 +1,194 @@
+import { describe, expect } from "vitest";
+import { createClient } from "@amqp-contract/client";
+import { createWorker } from "@amqp-contract/worker";
+import { it as baseIt } from "@amqp-contract/testing/extension";
+import { orderContract } from "./index.js";
+
+const it = baseIt.extend<{}>({});
+
+describe("Order Processing Integration", ({ amqpConnection }) => {
```

Comment:
we should split tests per concern : some tests only for client and some test only for worker

client and worker should carry their own tests and should not have any test in contrcat package